tests/integration/test_interaction_assessment.py

```diff
@@ -150,6 +160,86 @@ def test_max_attempts_reached_submit_and_reset_disabled(self):
         self.assertEqual(submit_button.get_attribute('disabled'), 'true')
         self.assertEqual(reset_button.get_attribute('disabled'), 'true')
 
+    def test_show_answer(self):
+        """
+        Test "Show Answer" button is shown in assessment mode, enabled when no
+        more attempts remaining, and displays correct answers when clicked.
+        """
+        show_answer_button = self._get_show_answer_button()
+        self.assertTrue(show_answer_button.is_displayed())
+
+        self.place_item(0, TOP_ZONE_ID)
+        for _ in xrange(self.MAX_ATTEMPTS):
+            self.assertEqual(show_answer_button.get_attribute('disabled'), 'true')
+            self.click_submit()
+
+        self.assertIsNone(show_answer_button.get_attribute('disabled'))
+        self.click_show_answer()
+
+        zones = dict(self.all_zones)
+        for item in self._get_items_with_zone(self.items_map).values():
+            zone_titles = [zones[zone_id] for zone_id in item.zone_ids]
+            # When showing answers, correct items are placed as if assessment_mode=False
+            self.assert_placed_item(item.item_id, zone_titles, assessment_mode=False)
+
+        for item in self._get_items_without_zone(self.items_map).values():
+            self.assertNotDraggable(item.item_id)
+            element = self._get_item_by_value(item.item_id)
```

Comment:
@arbrandes Nit: `element` is a bit generic. You could rename the loop variable to `item_definition`; that would make it possible to rename `element` to `item`.